### DISCUSSION TITLE
Watchdog function

### DISCUSSION THREAD

**Discussion by KlausJoergensen** [NicoHood/PinChangeInterrupt#42] [Q&A] [answered]
Hello,
I am using your software on an UNO with two wheel encoder. It runs perfect. But if the wheel is blocked, there is no watchdog function to recognize it. Is the arduino watchdog timer usable without problems? Do you have any other ideas?  

Kindly regards
Klaus

**Comment by NicoHood** [comment]
Which wheel? And what do you mean by blocked?

I am not aware that there should be issues related to the watchdog

**Comment by KlausJoergensen** [comment, marked as answer]
It is a two-wheel vehicle. If one of the wheels is blocked by an obstacle, the motor must stop or run backwards or else.
I was looking for something like 'no interrupt, then the wheel is not rolling'. Now I have found a solution by myself. Every encoder tick is setting a timeout. If a wheel is blocked, the tick is missing and the timeout stops the motor. 
Have a nice weekend
Klaus
